src/AppBundle/Resources/views/restaurant/form.html.twig

```diff
@@ -73,6 +73,7 @@
       {% if form.siret is defined %}
         {{ form_row(form.siret) }}
       {% endif %}
+      {{ form_row(form.refundableContainersEnabled) }}
```

Comment:
As this field is only available for admins, it should be wrapped inside `{% if form. refundableContainersEnabled is defined %}{% endif %}`